api/tests/Behat/bootstrap/v2/Common/INavigateToAdminTrait.php

```diff
@@ -73,4 +73,12 @@ public function iNavigateToTheReportChecklistPage()
         $this->clickLink('Checklist');
         $this->iAmOnAdminReportChecklistPage();
     }
+
+    /**
+     * @When I navigate to the admin report submissions page
+     */
+    public function iNavigateToAdminReportSubmissionsPage()
+    {
+        $this->clickLink('Submissions');
```

Comment:
It seems all the other functions have the `$this->iAmOnPage();` function to evaluate that you are indeed on the correct page after link click. Maybe want to do the same here?

Reply:
Good catch - will add in 👍